src/popup/pages/AccountDetails.vue

```diff
@@ -53,7 +55,10 @@
         name="fade-transition"
         mode="out-in"
       >
-        <RouterView :search-term="searchTerm" />
+        <RouterView
+          :search-term="searchTerm"
+          :display-filter="showSearchBar"
```

Comment:
I think we need to keep one way of naming the state which decides if something is displayed. Now once its `show` and second is `display`.